src/adv.py

```diff
@@ -54,32 +54,38 @@ def game():
     print('Game Options:')
     userInput = input("""[N] North [S] South [E] East [W] WEST [Q] Quit\n
     [(take) + (name of object)] Takes item [(drop) + (name of object)] Drops Item\n""").upper().split()
-    
+    print('#######################################################################################')
+
     while not userInput[0][0] == 'Q':
         
         if len(userInput) == 1:
-            userInput = userInput[0][0]
-            if (userInput == 'N') and (hasattr(player.room, 'n_to') == True):
+            if (userInput[0][0] == 'N') and (hasattr(player.room, 'n_to') == True):
                 player.room = player.room.n_to
-            elif userInput == 'S' and (hasattr(player.room, 's_to') == True):
+            elif userInput[0][0] == 'S' and (hasattr(player.room, 's_to') == True):
                 player.room = player.room.s_to
-            elif userInput == 'E' and (hasattr(player.room, 'e_to') == True):
+            elif userInput[0][0] == 'E' and (hasattr(player.room, 'e_to') == True):
                 player.room = player.room.e_to
-            elif userInput == 'W' and (hasattr(player.room, 'w_to') == True):
+            elif userInput[0][0] == 'W' and (hasattr(player.room, 'w_to') == True):
                 player.room = player.room.w_to
+            elif userInput[0] == 'I' or userInput[0] == 'INVENTORY':
+                print('\nMy Inventory:')
+                for item in player.inventory:
+                    print(item.name)
             else:
                 print('Could not move in that direction')
         elif len(userInput) == 2:
             targetItem = ''
-            if userInput[0] == 'TAKE':
+            if userInput[0] == 'TAKE' or userInput[0] == 'GET':
```

Comment:
watch your indentations and white spaces, code could look a lot cleaner if its at moderate